HISTORY.rst

```diff
@@ -2530,7 +2530,7 @@ Features
      @view_config(abc=1)
 
   Similar features exist for ``add_route``, and ``add_subscriber``.  See
-  "Adding A Third Party View, Route, or Subscriber Predicate" in the Hooks
+  "Adding A Custom View, Route, or Subscriber Predicate" in the Hooks
```

Comment:
I guess the reason is that HISTORY was included in docs but also in package long description (see https://github.com/Pylons/pyramid/pull/3563#discussion_r366588973).  I had changed my local copy to use `ref`, but it would be the only line in the file to do so, and render it dependent on Sphinx rather than pure reST, so I am reverting now (to the version with `Custom`, so that if anyone reads line 2533 they’ll have a chance at a successful docs search :slightly_smiling_face:)

Reply:
Yes the CHANGES is copied to HISTORY upon release of a major version. They should both minimize any linking to the rest of the document. The whatsnew docs will link back to them though to display the raw changelog.

Reply:
tox has a lint test for this as well: https://github.com/Pylons/pyramid/blob/master/tox.ini#L24